wg-batch/annual-report-2023.md

```diff
@@ -5,29 +5,31 @@
 
 1. What work did the WG do this year that should be highlighted?
 
-<!--
-   Some example items that might be worth highlighting:
-   - artifacts
-   - reports
-   - white papers
-   - work not tracked in KEPs
--->
+  Based on the discussions happening in our bi-weekly meetings we're continuing
+  collaboration with various stakeholders, the results are as follows:
+    - Promoted to beta [Allow Replacement of Pods in a Job when fully terminating](https://github.com/kubernetes/enhancements/tree/master/keps/sig-apps/3939-allow-replacement-when-fully-terminated).
+    - Promoted to beta [Backoff Limits Per Index For Indexed Jobs](https://github.com/kubernetes/enhancements/tree/master/keps/sig-apps/3850-backoff-limits-per-index-for-indexed-jobs).
+    - Introduced as beta [Elastic Indexed Job](https://github.com/kubernetes/enhancements/tree/master/keps/sig-apps/3715-elastic-indexed-job).
+    - Create [JobSet API for managing a group of k8s Jobs](https://github.com/kubernetes-sigs/jobset) and release [3 major versions in 2023](https://github.com/kubernetes-sigs/jobset/releases).
+    - [Kueue project](https://github.com/kubernetes-sigs/kueue/releases) released [3 major versions in 2023](https://github.com/kubernetes-sigs/kueue/releases).
```

Comment:
The list needs to have 2 spaces removed from the front each item, currently its rendering like this:

![Screenshot 2024-04-29 at 2 09 07 PM](https://github.com/kubernetes/community/assets/6500863/076fc91f-dd52-4c52-8c7a-b6eb7e7551ac)

```suggestion
  Based on the discussions happening in our bi-weekly meetings we're continuing
  collaboration with various stakeholders, the results are as follows:
  - Promoted to beta [Allow Replacement of Pods in a Job when fully terminating](https://github.com/kubernetes/enhancements/tree/master/keps/sig-apps/3939-allow-replacement-when-fully-terminated).
  - Promoted to beta [Backoff Limits Per Index For Indexed Jobs](https://github.com/kubernetes/enhancements/tree/master/keps/sig-apps/3850-backoff-limits-per-index-for-indexed-jobs).
  - Introduced as beta [Elastic Indexed Job](https://github.com/kubernetes/enhancements/tree/master/keps/sig-apps/3715-elastic-indexed-job).
  - Create [JobSet API for managing a group of k8s Jobs](https://github.com/kubernetes-sigs/jobset) and release [3 major versions in 2023](https://github.com/kubernetes-sigs/jobset/releases).
  - [Kueue project](https://github.com/kubernetes-sigs/kueue/releases) released [3 major versions in 2023](https://github.com/kubernetes-sigs/kueue/releases).
```

Reply:
Good catch - updated.